docs/resources/runbook.md

```diff
@@ -45,6 +45,7 @@ The following arguments are supported:
 * `type` - (Required) The type of the runbook. Valid values are 
   `incident`, `general`, `infrastructure`, and `incident_role`.
 * `description` - (Optional) A description of the runbook.
+* `owner_id` - (Optional) The ID of the team that owns this runbook.
```

Comment:
could you update the basic usage config to include owner_id? the service resource docs should have a team config you can copy